boom/command.py

```
@@ -1427,23 +1447,51 @@ def edit_entry(
     machine_id = machine_id or be.machine_id
     version = version or be.version
 
-    (add_opts_list, del_opts_list) = _merge_add_del_opts(be.bp, add_opts, del_opts)
+    add_opts_list: List[str] = []
+    del_opts_list: List[str] = []
+    if be.bp:
+        (add_opts_list, del_opts_list) = _merge_add_del_opts(be.bp, add_opts, del_opts)
+    elif add_opts or del_opts:
+        raise ValueError(
+            f"add_opts or del_opts given, but boot_id={be.boot_id} has no BootParams"
+        )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)